changelog.md

```diff
@@ -0,0 +1,15 @@
+# v0.8.0
```

Comment:
If you're calling this 2 do you also want to revise the version number to reflect that?